code/modules/lighting/lighting_object.dm

```diff
@@ -5,7 +5,7 @@
 
 	icon             = LIGHTING_ICON
 	icon_state       = "transparent"
-	color            = LIGHTING_BASE_MATRIX
+	color            = null //we manually set color in init instead
```

Comment:
Is this faster to set in `Initialize()` than here?

Reply:
Nah, that's not the problem.

The problem is that we have a system that creates lists of colors and their associated level of color protection on initialization. By setting the color, manually, after `. = ..()`, then we avoid thousands upon thousands of lists of color, thus cutting down on memory by a bit.

Reply:
I see